test/cpp/test_symint.cpp

```diff
@@ -129,5 +129,40 @@ TEST(SymintTest, TestDynamicSymints) {
   EXPECT_TRUE(si_element.GetSizeNode(2) != nullptr);
 }
 
+TEST(SymintTest, TestDynamicSymintArithmetic) {
+  torch::lazy::Value scalar_value =
+      torch::lazy::Value(ScalarOp(1.0, xla::F32), 0);
+  // Assign a incorrect 3d shape for the test purpose
+  std::vector<torch::lazy::Shape> abs_lazy_shapes = {
+      torch::lazy::Shape(torch::kFloat, {10, 20, 30})};
+  torch::lazy::NodePtr abs_node =
+      torch::lazy::MakeNode<Abs>(scalar_value, std::move(abs_lazy_shapes));
```

Comment:
fyi, this will break after I merge https://github.com/pytorch/xla/pull/3954 since now SizeNode checks xla::shape. check my test example in that pr. Using `expand` directly should be simpler.